fc-exploitation/views/identity-provider/update.ejs

```diff
@@ -505,9 +505,9 @@
             <div class="form-group row">
               <label class="col-2 col-form-label" for="userinfo_signed_response_alg">userinfo_signed_response_alg <%= !locals.instanceService.isCl() ? '*' : '' %> : </label>
               <div class="col-10">
-                <select class="custom-select" name="userinfo_signed_response_alg" <%= !locals.instanceService.isCl() ? 'required' : '' %>>
+                <select class="custom-select" name="userinfo_signed_response_alg">
```

Comment:
```suggestion
                <select class="custom-select" name="userinfo_signed_response_alg" required>
```

If it is not an instance of CoreLegacy, we add required, if it legacy we had nothing ('')